src/redux/reducers/rootReducer.js

```diff
@@ -0,0 +1,9 @@
+import { combineReducers } from 'redux';
+
+import locationReducer from '../reducers/locationReducer';
+import themeReducer from '../reducers/themeReducer';
+import weatherReducer from '../reducers/weatherReducer';
+
+const rootReducer = combineReducers({ locationReducer, themeReducer, weatherReducer });
```

Comment:
eventually you will have locationReducer etc names of properties on your state which is semantically weird. Did you mean to name them location, theme, weather ?